lib/zstd_stats.h

```diff
@@ -0,0 +1,164 @@
+/*
+    zstd - standard compression library
```

Comment:
I used this file for generation of stats (separately for literals, matches, offsets, ...). It might be useful in the future, but:
1. The file "zstd_stats.h" may not be released to public at all (more files = more confusion).
2. It's good to have the "zstd_stats.h" file in github (in case of e.g. HDD crash).